src/RMPI.jl

```diff
@@ -184,6 +184,90 @@ mutable struct MPIOSWin{T}  <: DistributeStrategy
 end
 Base.eltype(::MPIOSWin{T}) where T = T
 
+"""
+    MPIDefaultInplace{N,A}
+Like [`MPIDefault`](@ref), but uses preallocated buffers.
+"""
```

Comment:
It would be good to document the arguments of `MPIDefaultInplace()`, just to make the code more readable. The docstring might also mention to use the setup function `mpi_default_inplace()` for setup.